README.md

```diff
@@ -24,53 +23,23 @@
 
 ---
 
-#### **5)** We use `injectScript.js` to get the evolv remotecontext and send it to `evolvTool`. We actually have a bridge between webpage and `contentScript` to transfer data. We also can transfer data between `contentScript` and background as internal communication to make a complete data flow.
-
----
-
-#### **6)** Name the integration `EvoTools` and click the `Add New Connector` button at the bottom of the page.
-<img src="https://imgur.com/kBi2A9p.png"/>
-
----
-
-#### **7)** In the Connector Source dropdown, choose the `Gist` option.  In the `Gist` input, enter `153fb3d7cf8b170514343063cc2e43b5`.  Click the `Create` button in the top right corner.  If you don't see your integration appear in the list of integrations, refresh the page.
-<img src="https://imgur.com/p8lrM1H.png"/>
-
----
-
-#### **8)** Navigate to the `Environments` tab and click on the environment that you'd like to add the integration to.  When the side-drawer opens, click the `Integrations` tab followed by the `Add Integration` dropdown.  Choose `EvoTools` and then click `Save` in the top right corner.
-<img src="https://imgur.com/9n2qKcL.png"/>
-
----
-
-#### **9)** You're all set up!  Navigate to a page that you know has an active experiment, click the extension icon, and validate that you are seeing data.
+#### **5)** You're all set up!  Navigate to a page that you know has an active experiment, click the extension icon, and validate that you are seeing data.
 <img src="https://imgur.com/AHn9ubo.png"/>
 
-
 ---
----
-
 
-This extension is still in the very early stages and hasn't been widely used so please report any issues or feature requests to brian.norman@evolv.ai.
+### Note:
+We use `injectScript.js` to get the evolv remotecontext and send it to `evolvTool`. We actually have a bridge between webpage and `contentScript` to transfer data. We also can transfer data between `contentScript` and background as internal communication to make a complete data flow.
 
 ---
 
-## Possible Names for Extension:
-* Evolv Debugger
-* Evolv Assistant
-* EvoEye
-* Evolv Aeye
-* EvoSpy
-* Evolv Inspector
-* Inspector Evo
+This extension is still in the very early stages and hasn't been widely used so please submit an issue when you got one.
```

Comment:
I'd delete this -- just because we'll forget to remove it in a year when it's no longer true